src/test/test-bitfield.c

```diff
@@ -213,6 +213,8 @@ TEST(bits) {
         assert_se(_u64 == u64);
 
         /* Verify these use cases are constant-folded. */
+#if !defined(__clang__) || (__clang_major__ >= 13)
```

Comment:
Why not `>= 12` ? Or Clang 12 also does not like the tests?

Reply:
Here are some links to the compiler explorer:

- clang-13 (compiles): https://godbolt.org/z/n4xveaxG8
- clang-12 (fails): https://godbolt.org/z/Kn8jnqx3x

So it seems that clang version 13 is the minimum version.

Reply:
-> #28601